Runmon/44_GauntletPlatinum.yaml

```
@@ -0,0 +1,215 @@
+species: Lumineon
+ingamename: "AACm   neg"
+displayname: "AACm   neg"
+gender: [m]
+setname: TPP Gauntlet Platinum
+tags: [runmon, in-game]
+item: [Sea Incense]
+ability: [Swift Swim]
+moves:
+    - [Surf (=18)]
+    - [Waterfall]
+    - [Ice Beam]
+    - [Rain Dance]
+happiness: 254
+biddable: True
+rarity: 1.0
+ball: [Great]
+shiny: False
+nature: Docile
+ivs: {hp: 27, atk: 26, def: 15, spA: 26, spD: 24, spe: 15}
+evs: {hp: 31, atk: 150, def: 158, spA: 57, spD: 23, spe: 91}
+---
+species: Altaria
+ingamename: "!!    "
+displayname: "!!    "
+gender: [f]
+setname: TPP Gauntlet Platinum
+tags: [runmon, in-game]
+item: [Sky Plate]
+ability: [Natural Cure]
+moves:
+    - [Fly]
+    - [Fire Blast]
+    - [Earthquake]
+    - [Dragon Dance]
+happiness: 255
+biddable: True
+rarity: 1.0
+ball: [Great]
+shiny: False
+nature: Sassy
+ivs: {hp: 6, atk: 16, def: 14, spA: 5, spD: 19, spe: 17}
+evs: {hp: 27, atk: 172, def: 111, spA: 85, spD: 35, spe: 80}
+---
+species: Leafeon
+ingamename: "???') !"
+displayname: "☺☺☺') !"
+gender: [m]
+setname: TPP Gauntlet Platinum
+tags: [runmon, in-game]
+item: [null]
+ability: [Leaf Guard]
+moves:
+    - [Razor Leaf]
+    - [Giga Drain]
+    - [Strength]
+    - [Synthesis]
+happiness: 255
+biddable: True
+rarity: 1.0
+ball: [Poké]
+shiny: False
+nature: Docile
+ivs: {hp: 27, atk: 29, def: 26, spA: 30, spD: 13, spe: 18}
+evs: {hp: 50, atk: 111, def: 125, spA: 66, spD: 30, spe: 100}
+---
+species: Weavile
+ingamename: "AAAL"
+displayname: "AAAL"
+gender: [m]
+setname: TPP Gauntlet Platinum
+tags: [runmon, in-game]
+item: [Exp. Share]
+ability: [Pressure]
+moves:
+    - [Ice Punch]
+    - [Night Slash]
+    - [Brick Break]
+    - [Shadow Claw]
+happiness: 254
+biddable: True
+rarity: 1.0
+ball: [Repeat]
+shiny: False
+nature: Lonely
+ivs: {hp: 0, atk: 25, def: 3, spA: 9, spD: 23, spe: 28}
+evs: {hp: 25, atk: 116, def: 86, spA: 70, spD: 29, spe: 106}
+---
+species: Probopass
+ingamename: "GUUEEEEFFn"
+displayname: "GUUEEEEFFn"
+gender: [m]
+setname: TPP Gauntlet Platinum
+tags: [runmon, in-game]
+item: [null]
+ability: [Sturdy]
+moves:
+    - [Thunder Wave]
+    - [Flash Cannon]
+    - [Power Gem]
+    - [Discharge]
+happiness: 255
+biddable: True
+rarity: 1.0
+ball: [Poké]
+shiny: False
+nature: Mild
+ivs: {hp: 25, atk: 7, def: 17, spA: 11, spD: 28, spe: 22}
+evs: {hp: 36, atk: 140, def: 100, spA: 58, spD: 22, spe: 132}
+---
+species: Torterra
+ingamename: "BFG"
+displayname: "BFG"
+gender: [m]
+setname: TPP Gauntlet Platinum
+tags: [runmon, in-game]
+item: [Razor Claw]
+ability: [Overgrow]
+moves:
+    - [Razor Leaf]
+    - [Curse]
+    - [Synthesis, Earthquake]
+    - [Rock Climb, Earthquake]
```

```suggestion
    - [Rock Climb, Earthquake]
separations:
    - [Earthquake, Earthquake]
```
Have to do this so it doesn't end up with EQ in both slots.